repository/Grease-Tests-Core.package/GRPlatformTest.class/instance/runCase.st

```diff
@@ -0,0 +1,3 @@
+running
+runCase
+	GRPlatform current doSilently: [ super runCase ]
```

Comment:
Hey @marschall 
Question: why is this needed and how does it work? the `doSilently:` does nothing but evaluation the block?

Reply:
> Hey @marschall
> Question: why is this needed and how does it work? the `doSilently:` does nothing but evaluation the block?

That's only the default implementation. On Pharo it suspends the announcements preventing the package from becoming dirty

https://github.com/SeasideSt/Grease/blob/master/repository/Grease-Pharo100-Core.package/GRPharoPlatform.class/instance/doSilently..st

This is used by `WAFileLibraryTest` where the `ByteArray` equivalent is tested

https://github.com/SeasideSt/Seaside/blob/master/repository/Seaside-Tests-Pharo-Core.package/WAFileLibraryTest.extension/instance/runCase.st

Reply:
Sorry, I should have read this in an image and I would have realised this is not a new method. Now I was just reading the changes on github and assumed this was newly introduced.